EEBOS/owl-ui
# [sc-852] - Feature/Tabs Component

### Short summary

Created default and dark variants of tabs component with accessibility considerations. Still looking into refactor for additional typescript. May push up a couple additional updates.

---

### Test steps

1. Run `yarn start:storybook`
2. Check out the Tabs component on Storybook and ensure the different versions work properly.
